### PR TITLE
Install branch built v2plugin

### PR DESCRIPTION
**Pull request by chrisplo**
Works with:
https://github.com/contiv/ansible/pull/382

env.json now passes a flag to build the v2plugin at install time or
not, and that flag is managed during build.sh based on if
NETPLUGIN_BRANCH is set during build.

All the scripts and resources in netplugin:install/v2plugin are copied
to the install archive's staging directory then into the installer
archive.

renamed prepare_netplugin_tarball.sh to prepare_netplugin_artifacts.sh
because it does more than just prepare a tarball now

Drive-by:
* Bumps the default contiv version to 1.1.7
* v2plugin version defaults to the default contiv version

Signed-off-by: Chris Plock <chrisplo@cisco.com>

### CODE REVIEW

**Comment by chrisplo** [comment]
works with https://github.com/contiv/ansible/pull/382

File: Makefile [hunk 1]
```diff
@@ -5,20 +5,22 @@ export CONTIV_INSTALLER_VERSION ?= $(BUILD_VERSION)
...
 export CONTIV_NETPLUGIN_VERSION := $(NETPLUGIN_OWNER)-$(NETPLUGIN_BRANCH)
+export CONTIV_V2PLUGIN_VERSION ?= $(NETPLUGIN_OWNER)-$(NETPLUGIN_BRANCH)
 endif
-export CONTIV_V2PLUGIN_VERSION ?= $(DEFAULT_DOWNLOAD_CONTIV_VERSION)
 export CONTIV_NETPLUGIN_TARBALL_NAME := netplugin-$(CONTIV_NETPLUGIN_VERSION).tar.bz2
-export CONTIV_ANSIBLE_COMMIT ?= 8e20f56d541af8bc7a3ecbde0d9c64fa943812ed
-export CONTIV_ANSIBLE_OWNER ?= contiv
+export CONTIV_V2PLUGIN_TARBALL_NAME := v2plugin-$(CONTIV_V2PLUGIN_VERSION).tar.gz
+export CONTIV_ANSIBLE_COMMIT ?= build_v2plugin_on_demand
+export CONTIV_ANSIBLE_OWNER ?= chrisplo
```
Comment (tiewei) [inline Makefile]: roll these two lines back when https://github.com/contiv/ansible/pull/382 merged

**Comment by chrisplo** [comment]
build PR

**Comment by chrisplo** [comment]
the ansible dependency landed, I need to update this to point to current master

**Comment by chrisplo** [comment]
build PR

File: Makefile (hunk 1 above)
Comment (chrisplo) [reply Makefile]: done

**Pull request merged**